sonarqube/ocp-config/sonarqube.yml

```diff
@@ -380,46 +380,40 @@ objects:
     sessionAffinity: None
     type: ClusterIP
 parameters:
-- displayName: OC project
-  name: OC_PROJECT
-  description: Name of the ocp project to deploy to. This is normally the "cd" project
-  required: true
-# this must be base64 encoded.
 - displayName: Admin Password
-  name: ADMIN_PASSWORD
+  name: SONAR_ADMIN_PASSWORD_B64
```

Comment:
I think making it explicit that these are in base64 is a good thing.

However the tailor .B64 behavior for secrets could cause confusion here. 
See https://github.com/opendevstack/tailor/issues/148#issuecomment-548367383 also.

Reply:
Yea - right now we do not use Tailor's encryption feature for ODS anyway though.